docs/user/integration/dynatrace/exporter-values.yaml

```diff
@@ -39,8 +40,20 @@ config:
         statements:
           # convert the <histogram_name>_sum metrics to gauges.
           - convert_sum_to_gauge() where IsMatch(metric.name, ".*_sum")
+      - context: metric
+        statements:
+          # Remove empty resource attributes
+          - delete_key(resource.attributes, "k8s.node.name") where resource.attributes["k8s.node.name"] == ""
+          - delete_key(resource.attributes, "k8s.namespace.name") where resource.attributes["k8s.namespace.name"] == ""
+          - delete_key(resource.attributes, "k8s.cluster.name") where resource.attributes["k8s.cluster.name"] == ""
```

Comment:
The cluster name cannot be empty. If it is the case, we must fix it..

Reply:
removed